### DISCUSSION TITLE
How to insesrt transformedimages?

### DISCUSSION THREAD

**Discussion by anvme** [spacecatninja/craft-imager-x#238] [Q&A]
In the config, I have a few image transformations. How to insert them in the template?
I have FeedbackTransformThumbnail, but the documentation is poor, and there is no any information about this case. Only transformation

**Comment by aelvan** [comment]
Please refer to https://imager-x.spacecat.ninja/usage/.

The first (Imager) example of that page explains how to use a single transformed image:

```
{% set image = craft.assets().one() %}
{% set transformedImage = craft.imagerx.transformImage(image, { width: 1000 }) %}
<img src="{{ transformedImage.url }}">
```

The page proceeds to explain that Imager is geared towards creating a range of transforms, that will be returned as an array to the template, and can for instance be passed to the srcset helper method in Imager to be used as a source set (from the following examples):

```
{% set transformedImages = craft.imagerx.transformImage(image, [
    { width: 1200 }, 
    { width: 1000 }, 
    { width: 800 }, 
    { width: 600, jpegQuality: 65 }, 
    { width: 400, jpegQuality: 65 }
    ], { ratio: 16/9, position: '100% 100%', jpegQuality: 80 }) %}
    
<img src="{{ craft.imagerx.placeholder({ width: 160, height: 90 }) }}" 
     sizes="100vw" 
     srcset="{{ craft.imagerx.srcset(transformedImages) }}">    
```

But you're free to use the returned data any way you want. It's either a single model, or an array of models, with properties as describe in [the Models section of the docs](https://imager-x.spacecat.ninja/models.html#crafttransformedimagemodel).
